cnd/tests/bitcoin_helper/connector_mock.rs

```diff
@@ -34,15 +34,15 @@ impl BitcoinConnectorMock {
 }
 
 impl LatestBlock for BitcoinConnectorMock {
-    type Error = ();
+    type Error = String;
```

Comment:
This is Rust, let's use types :)

Reply:
If this builds I'm happy, I was just keeping the compiler quiet :)

Reply:
Dealt with this in https://github.com/comit-network/comit-rs/pull/1849.